test/tests/ver-1kc/tests

```diff
@@ -5,14 +5,27 @@
   [ver-1kc_csv]
     type = CSVDiff
     input = ver-1kc.i
+    cli_args = "nb_segments_TMAP8='${fparse 1e3}'
+                Executioner/nl_abs_tol='${fparse 1e-3}'
+                Executioner/nl_rel_tol='${fparse 1e-2}'
```

Comment:
```suggestion
    cli_args = "nb_segments_TMAP8=1e1
                simulation_time=4
                Executioner/nl_abs_tol=1e-8
                Executioner/nl_rel_tol=1e-6
```
Why do you use an even finer mesh here? The goal is to have a fast-running simulation. 
The tolerance you have here is too large. I'm sure you can refine this and still run under 2 seconds. Especially with the coarser mesh. 
You can also run the test for a shorter time.